coops-ui/src/interfaces/informationAlert-interfaces.ts

```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue'
+
+export interface AlertMessage extends Vue {
```

Comment:
This doesn't extend Vue, so you can just write `export interface AlertMessage {` (and stop importing Vue). I know you just copied what was there before but it's wrong too ... feel free to update the other interfaces.

This file contains a single interface, so I'd be tempted to name it singular instead of plural.

Why not name this file alertMessage-interface, according to its content?

(Sorry, I'm picky today).

Reply:
I've gone with plural because, conceivably, there could be more informational alert types